tests/features/ecas-login.feature

```diff
@@ -1,16 +1,22 @@
-@api @javascript @ecas-login
+@api @javascript @ecas-login @casMockServer
```

Comment:
Is this tag (and its related function \Drupal\Tests\oe_authentication\Behat\AuthenticationContext::resetAuthenticationMock) still needed? Javascript tag is not needed.
```suggestion
@api @casMockServer
```